ui/server/develConf/sample/users.conf

```diff
@@ -2,13 +2,8 @@ users: [
   {
     id: "admin"
     password: "admin"
-    categoryPermissions: {
-      "Category1": ["Read", "Write", "Deploy", "Admin"]
-      "Category2": ["Read", "Write", "Deploy", "Admin"]
-      "Technical": ["Read", "Write", "Deploy", "Admin"]
-      "Default": ["Read", "Write", "Deploy", "Admin"]
-      "StandaloneCategory1": ["Read", "Write", "Deploy", "Admin"]
-    }
+    isAdmin: true
```

Comment:
I think we should fix it also in different places - e.g. docker-users.conf, etc?

Reply:
Ofc you are right :)